<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signin</class>
 <widget class="QDialog" name="Signin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(85, 85, 127);</string>
  </property>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	background-color:transparent;
	border-radius:8px;

}
QPushButton:hover
{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 85, 0, 255), stop:1 rgba(255, 0, 255, 255));
	border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/closeWHITE</normaloff>:/icon1/closeWHITE</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="complete">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:10px;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.999316, y1:0.353, x2:1, y2:0, stop:1 rgba(57, 166, 255, 255));
}
QPushButton:hover
{
	border-radius:5px;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.905, y1:0.159682, x2:0.973684, y2:0.96, stop:0.378947 rgba(6, 178, 255, 255), stop:1 rgba(255, 255, 255, 255))
}
QPushButton:disabled
{
	border-radius:10px;
	color:black;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(163, 163, 163, 255), stop:1 rgba(173, 173, 173, 255));
}
QPushButton:press
{
	border-radius:2px;
	background-color:transparent;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.905, y1:0.159682, x2:0.973684, y2:0.96, stop:0.378947 rgba(21, 178, 255, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
      </property>
      <property name="text">
       <string>完成</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="completeandenter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:10px;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.999316, y1:0.353, x2:1, y2:0, stop:1 rgba(57, 166, 255, 255));
}
QPushButton:hover
{
	border-radius:5px;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.905, y1:0.159682, x2:0.973684, y2:0.96, stop:0.378947 rgba(6, 178, 255, 255), stop:1 rgba(255, 255, 255, 255))
}
QPushButton:disabled
{
	border-radius:10px;
	color:black;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(163, 163, 163, 255), stop:1 rgba(173, 173, 173, 255));
}
QPushButton:press
{
	border-radius:2px;
	background-color:transparent;
	color:white;
	font-size:17px;
	background:qlineargradient(spread:pad, x1:0.905, y1:0.159682, x2:0.973684, y2:0.96, stop:0.378947 rgba(21, 178, 255, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
      </property>
      <property name="text">
       <string>完成并进入</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupbox2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color:transparent;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="inputtips">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 0px;
	border-radius: 15px;
	background-color:transparent;
	color: white;
}</string>
    </property>
    <property name="placeholderText">
     <string>设置提示信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tipicon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/proofWHITE</normaloff>
      <disabledoff>:/icon1/proofBLUE</disabledoff>:/icon1/proofWHITE</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="tipline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 2px solid white; 
background-color:transparent;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputanswer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 0px;
	border-radius: 15px;
	background-color:transparent;
	color: white;
}</string>
    </property>
    <property name="placeholderText">
     <string>设置答案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ansicon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/spawdWHITE</normaloff>
      <disabledoff>:/icon1/spswdBLUE</disabledoff>:/icon1/spawdWHITE</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="ansline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 2px solid white; 
background-color:transparent;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color: white;</string>
    </property>
    <property name="text">
     <string>完善验证信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/helpWHITE</normaloff>
      <disabledoff>:/icon1/helpBLUE</disabledoff>:/icon1/helpWHITE</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color:transparent;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="Line" name="pswdline">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 2px solid white; 
background-color:transparent;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 0px;
	border-radius: 15px;
	background-color:transparent;
	color: white;
}</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="placeholderText">
     <string>设置密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pswdicon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/pswdWHITE</normaloff>
      <disabledoff>:/icon1/pswdBLUE</disabledoff>:/icon1/pswdWHITE</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pswdhelp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/helpWHITE</normaloff>:/icon1/helpWHITE</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pswdwarning">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/rightGREEN</normaloff>
      <disabledoff>:/icon1/warningRED</disabledoff>:/icon1/rightGREEN</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 0px;
	border-radius: 15px;
	background-color:transparent;
	color: white;
}</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="placeholderText">
     <string>设置用户名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="usericon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/usernameWHITE</normaloff>
      <disabledoff>:/icon1/usernameBLUE</disabledoff>:/icon1/usernameWHITE</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="userline">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 2px solid white; 
background-color:transparent;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="userhelp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px;
	background-color:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/helpWHITE</normaloff>:/icon1/helpWHITE</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="userwarning">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon1/rightGREEN</normaloff>
      <disabledoff>:/icon1/warningRED</disabledoff>:/icon1/rightGREEN</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	background-color:transparent;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon1/nextGREEN</normaloff>:/icon1/nextGREEN</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
